llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -1232,6 +1396,12 @@ def flops_per_batch(self, batch: Mapping):
         # that the dataset has been constructed without padding. Additionally, we
         # assume the backward pass is approximately 2x the forward pass
 
+        if self.model.config.block_overrides is not None:
```

Comment:
Can you add a jira ticket to update this?
(and also I'm pretty sure we never updated this for swa... 😬 )

Reply:
Done.